src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -439,6 +442,9 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
     }
 
     correctColumns() {
+        if (this.temporalProperties.controls['isTime'].disabled) {
```

Comment:
So columns only get corrected if some control is disabled?

Reply:
correctColumns method ensures that the temporal columns don't collide with the spatial columns. Therefore when isTime is disabled it is not necessary to correct anything. Further since the temporal-columns would start out of range for 2-column csvs this method could get stuck because there is no place to actually put the temporal columns.

Reply:
It returns if it is disabled, meaning that columns only get corrected if the control is enabled.